src/operations.rs

```diff
@@ -123,10 +123,10 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
-    unsafe {
-        let value = transmute!(value);
-        xor ^ transmute::<_, u128>(vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))))
-    }
+    let value = transmute!(value);
+    let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
```

Comment:
Optionally, you could alternatively inline here (I know I suggested the current text lol):

```suggestion
    let res = unsafe { vaesmcq_u8(vaeseq_u8(transmute!(value), transmute!(0u128))) };
```